updating/updating-cluster-prepare.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="updating-cluster-prepare"]
+= Preparing to update to {product-title} 4.9
+include::modules/common-attributes.adoc[]
+:context: updating-cluster-prepare
+
+toc::[]
+
+{product-title} 4.9 uses Kubernetes 1.22, which removed a significant number of deprecated `v1beta1` APIs.
+
+{product-title} 4.8.14 introduced a requirement that an administrator must provide a manual acknowledgment before the cluster can be upgraded from {product-title} 4.8 to 4.9. This is to help prevent issues after upgrading to {product-title} 4.9, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this evaluation and migration is complete, the administrator can provide the acknowledgment.
+
+Before you can upgrade your {product-title} 4.8 cluster to 4.9, you must provide the administrator acknowledgment.
```

Comment:
This is my issue, I think, but I haven't been able to find this sentence in the preview output. 

As far as I can tell, I've found the named file and I found the rest of this text, but not this sentence. I'm sure it's there somewhere. :-(